pkg/indexer/storer/identityUpdate.go

```diff
@@ -35,21 +50,78 @@ func (s *IdentityUpdateStorer) StoreLog(ctx context.Context, event types.Log) Lo
 		return NewLogStorageError(err, false)
 	}
 
-	// TODO:nm figure out topic structure
+	latestSequenceId, err := s.queries.GetLatestSequenceId(ctx, IDENTITY_UPDATE_ORIGINATOR_ID)
+	if err != nil {
+		return NewLogStorageError(err, true)
+	}
+	if uint64(latestSequenceId) >= msgSent.SequenceId {
```

Comment:
Should we log a warning or error here? I assume in the future we will have a mechanism to start syncing from the correct location?

Reply:
Going to add that mechanism next.